docs/source/quick_start.rst

```diff
@@ -0,0 +1,175 @@
+    2018-08-07 00:58:56,888 - maestrowf.datastructures.core.study:add_step:307 - INFO - post-process-lulesh is dependent on run-lulesh_*. Creating edge (run-lulesh_*, post-process-lulesh)...
+    2018-08-07 00:58:56,889 - maestrowf.datastructures.core.study:add_step:298 - INFO - Adding step 'post-process-lulesh-trials' to study 'lulesh_sample1'...
+    2018-08-07 00:58:56,889 - maestrowf.datastructures.core.study:add_step:307 - INFO - post-process-lulesh-trials is dependent on run-lulesh_*. Creating edge (run-lulesh_*, post-process-lulesh-trials)...
+    2018-08-07 00:58:56,889 - maestrowf.datastructures.core.study:add_step:298 - INFO - Adding step 'post-process-lulesh-size' to study 'lulesh_sample1'...
+    2018-08-07 00:58:56,889 - maestrowf.datastructures.core.study:add_step:307 - INFO - post-process-lulesh-size is dependent on run-lulesh_*. Creating edge (run-lulesh_*, post-process-lulesh-size)...
+    2018-08-07 00:58:56,890 - maestrowf.datastructures.core.study:setup_workspace:337 - INFO - Setting up study workspace in '/home/travis/build/LLNL/maestrowf/testing/lulesh'
+    2018-08-07 00:58:56,890 - maestrowf.datastructures.core.study:setup_environment:347 - INFO - Environment is setting up.
+    2018-08-07 00:58:56,890 - maestrowf.datastructures.core.studyenvironment:acquire_environment:191 - INFO - Acquiring dependencies
+    2018-08-07 00:58:56,890 - maestrowf.datastructures.core.studyenvironment:acquire_environment:193 - INFO - Acquiring -- LULESH
+    2018-08-07 00:58:56,890 - maestrowf.datastructures.environment.gitdependency:acquire:150 - INFO - Cloning LULESH from https://github.com/LLNL/LULESH.git...
+    Cloning into '/home/travis/build/LLNL/maestrowf/testing/lulesh/LULESH'...
+    remote: Counting objects: 64, done.
+    remote: Compressing objects: 100% (40/40), done.
+    remote: Total 64 (delta 10), reused 43 (delta 6), pack-reused 16
+    Unpacking objects: 100% (64/64), done.
+    2018-08-07 00:58:57,306 - maestrowf.datastructures.core.study:configure_study:382 - INFO -
+    ------------------------------------------
+    Output path =               /home/travis/build/LLNL/maestrowf/testing/lulesh
+    Submission attempts =       1
+    Submission restart limit =  1
+    Submission throttle limit = 0
+    Use temporary directory =   False
+    ------------------------------------------
+    2018-08-07 00:58:57,307 - maestrowf.datastructures.core.executiongraph:__init__:337 - INFO -
+    ------------------------------------------
+    Submission attempts =       1
+    Submission throttle limit = 0
+    Use temporary directory =   False
+    Tmp Dir =
+    ------------------------------------------
+
+Once set up is complete, Maestro will begin expanding the Study's graph into an ``ExecutionGraph``. The ``ExecutionGraph`` represents the complete execution plan for a study. The snippets below show some of the expected output which will be a mix of single steps and parameterized steps.

```

Comment:
Probably it would be good to include something like the following: 
"After study expansion, the ``ExecutionGraph`` will persist for the remainder of the study, and will be accessed by a daemon process called the ``Conductor`` that monitors the study's execution."